src/commands/edge/drivers/logcat.ts

```diff
@@ -105,6 +111,16 @@ export default class LogCatCommand extends SseCommand<typeof LogCatCommand.flags
 		},
 	]
 
+	private async getMinLogLevel(): Promise<number> {
```

Comment:
This method doesn't have any reason be `async`. It could just return `number`.

Reply:
Good catch.  Thanks!